src/main/java/com/example/damian/iracki/ticketBookingApp/enums/TypeOfTicket.java

```diff
@@ -0,0 +1,19 @@
+package com.example.damian.iracki.ticketBookingApp.enums;
+
+import java.math.BigDecimal;
+
+public enum TypeOfTicket {
+    NORMAL(new BigDecimal(25)),
+    STUDENT(new BigDecimal(15)),
+    CHILD(new BigDecimal("12.5"));
```

Comment:
Tutaj tworzysz `BigDecimal` ze `Stringa`, a wyżej bezpośrednio z `int`. Skąd taki wybór? Czy mógłbyś napisać po prostu `new BigDecimal(12.5)`?

Reply:
Dla wartości zmiennoprzecinkowych rekomendowany jest konstruktor wykorzystujący String.
Konstruktory wykorzystujące typy zmiennoprzecinkowe nie określają jednoznacznie poziomu precyzji. Ustawiają go na wartość specyficzną dla danego typu. Może przez to dojść do utraty precyzji.